src/models/DataRecord.model.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema
+var autoIncrement = require('mongoose-plugin-autoinc');
+
+var DataSchema = new Schema({
+	'time': String,
+	'completionID': String,
+	'experimentID': Number,
+	'configID': Number,  //foreign key to Configs collection?
+	'data': String,
```

Comment:
Would it make sense to store data as a `mongoose.Mixed` type (Object type) instead of a stringified JSON? 
Not sure on this one, would appreciate your thoughts.
https://mongoosejs.com/docs/schematypes.html#mixed